prisoner-content-hub-frontend.env

```diff
@@ -0,0 +1,24 @@
+PORT=3000
+
+# The following must match the container names in your docker-compose or local setup
+HUB_API_ENDPOINT=http://drupal:8080
```

Comment:
I think maybe we'd want the default to be the Drupal development environment?  And then a way to optionally override this with a local version of Drupal?  Upto you @BarryGeeMOJ